exercises/practice/piecing-it-together/.meta/config.json

```
@@ -0,0 +1,22 @@
+{
+  "authors": [
+    "zamora-carlos"
+  ],
+  "files": {
+    "solution": [
+      "src/main/java/PiecingItTogether.java"
+    ],
+    "test": [
+      "src/test/java/PiecingItTogetherTest.java"
+    ],
+    "example": [
+      ".meta/src/reference/java/PiecingItTogether.java"
+    ],
+    "editor": [
+      ".meta/src/reference/java/JigsawInfo.java"
```

This should point to the `src/main/java` one.

```suggestion
      "src/reference/java/JigsawInfo.java"
```